core/lib/server/pub.ts

```diff
@@ -1059,14 +1068,20 @@ type MaybePubChildren<Options extends GetPubsWithRelatedValuesAndChildrenOptions
  * Only add the `stage` if the `withStage` option has not been set to `false
  */
 type MaybePubStage<Options extends GetPubsWithRelatedValuesAndChildrenOptions> =
-	Options["withStage"] extends true ? { stage: Stages } : { stage?: never };
+	Options["withStage"] extends true ? { stage?: Stages } : { stage?: never };
 
 /**
  * Only add the `pubType` if the `withPubType` option has not been set to `false
  */
 type MaybePubPubType<Options extends GetPubsWithRelatedValuesAndChildrenOptions> =
 	Options["withPubType"] extends true ? { pubType: PubTypes } : { pubType?: never };
 
+/**
+ * Only add the `pubType` if the `withPubType` option has not been set to `false
+ */
+type MaybePubMembers<Options extends GetPubsWithRelatedValuesAndChildrenOptions> =
+	Options["withMembers"] extends true ? { members: Users[] } : { members?: never[] };
+
```

Comment:
just a headsup that all this was moved to `contracts` recently so this might be an annoying merge